dashboard/config/locales/en.yml

```diff
@@ -1402,3 +1402,9 @@ en:
     thank_you: "THANK YOU for championing computer science education. Our team wishes you and your students an unforgettable Hour of Code!"
     founder: "Founder, Code.org"
     your_friends: "Your Friends at Code.org"
+  lti:
+    error:
+      missing_params: "Missing required param(s): client_id, lms, email"
+      integration_exists: "Lti Integration already exists for this Client ID"
+      wrong_resource_type: "Only LtiResourceLink is supported right now"
+    create_integration_success: "Succesfully created your LTI integration"
```

Comment:
General question about naming convention here. Do these keys need to be unique if they are namespaced like they are here? For example, there is another `missing_params` key in this file, but it is namespaced under `lesson_lock.error.missing_params`.

Reply:
Not sure but it seems like duplicate keys should be fine if they're at different levels of the hierarchy

Reply:
This looks great!